torch/quantization/fuse_modules.py

```diff
@@ -3,6 +3,15 @@
 import torch
 
 import torch.nn._intrinsic.modules.fused as torch_fused
+def fuse_conv_relu(conv, relu):
+    r"""Given the conv and relu modules, fuses them and returns the fused module
+    """
+    return torch.nn._intrinsic.ConvReLU2d(conv, relu)
+
+def fuse_linear_relu(linear, relu):
+    r"""Given the linear and relu modules, fuses them and returns the fused module
+    """
+    return torch.nn._intrinsic.LinearReLU(linear, relu)
 
 def fuse_conv_bn(conv, bn):
```

Comment:
this function (and probably `fuse_conv_bn_eval`) misses setting the training mode. Something like `result.train(original.training)`

Reply:
There is no need to do that here, we just create a sequential here with the original modules, so all attributes are preserved